src/index.ts

```diff
@@ -90,8 +126,34 @@ const getRepositories = async (octokit: Octokit, login: string): Promise<Reposit
     repositories.push(...page.nodes);
   }
   console.log('Found ' + repositories.length + ' repositories.')
-  return repositories;
-}
+
+  // A function that allows the user to filter the repositories
+  const getSelections: (repos: Repository[]) => Promise<Repository[]> = async (repos) => {
```

Comment:
nit: It's a bit odd to have this function nested within the other. It might be simpler to extract the filtering part to an extra step that is just called directly from the main function near the bottom (where `getRepositories` itself is called)